### ISSUE TITLE
A11y_.NET Core_Recipes App My Recipes New_New Recipe_Screen reader: After changing the value for review, Screen reader is not announcing new value its announcing incorrect rating value.

### ISSUE THREAD

**Issue by Swati1700** [dotnet/maui#15741]
### Description

[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[MAUI]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#Benchmark;#Narrator;#NVDA;#Element:Label

### Actual Results:
In "New Recipe" screen reader is announcing incorrect information. After changing the value for review, screen reader is not announcing new value its announcing incorrect rating value like "10.0 out of 10".

**Similar issue is also reproducing in the below scenarios:
Issue: Recipes App My Recipes New_New Recipe**

1. Install and open ".NET MAUI" app.
2. TAB till "My Recipes" and select.
3. TAB till any existing recipe and select.
4. "Recipe" Screen will open.
5. TAB till "Edit" button and select.
6. Enter text in all entry fields.
7. Start Screen reader.
8. TAB till Rating value.
9. Observe Screen reader is not announcing correct rating after updating the review rating.

### Expected Results:
After changing the value for review, Screen reader should announce new updated rating value like "7.3 out of 10".

### Steps to Reproduce:
1.Install and open ".NET MAUI" app.
2.TAB till "My Recipes" and select.
3.TAB till "New" button and select.
4."New Recipe" Screen will open.
5. Start Screen reader.
6. TAB till Slider and increment, decrement button and do changes in review.
7. TAB till rating displayed above the slider.
8. Observe voiceover is not announcing correct rating after updating the review rating.

### User Impact:
Screen reader users will face difficulties while navigating to the rating value if screen reader will announce incorrect announcement of rating.

### Link to public reproduction project repository

Test

### Version with bug

7.0.49

### Last version that worked well

6.0

### Affected platforms

Windows

### Affected platform versions

Windows 11 23H2

**Comment by rachelkang** [comment]
Hi, @Swati1700 - could you please provide clarification and/or video for the expected vs. actual behaviors?

To my understanding, we already match the expected behavior mentioned here already aligns with the actual behavior.
When updating the slider, Narrator announces something like "73, Rating decreased to 7.3 out of 10 stars"

The "73" is the slider value as a percentage. This behavior is external, and is announced as a consequence of the WinUI platform behavior. The latter "Rating decreased to 7.3 out of 10 stars" is what we've added to reflect the expected behavior you describe.

Please let me know if I'm missing something here. Thanks!

**Comment by Swati1700** [comment]
Verified the issue and now NVDA and narrator is announcing expected announcement hence closing this bug please find the closing attachment below.

https://github.com/dotnet/maui/assets/86710888/182f06ba-ea70-48f5-938d-a927acb13239

![Screenshot (1304)](https://github.com/dotnet/maui/assets/86710888/54e32414-d11d-4039-b8dc-57678b170376)
